script/todo.php

```diff
@@ -103,10 +103,12 @@ protected function content(array $values): string
         }
 
         $content = implode($this->eol, $values);
+        $sumMissing = count($values);
```

Comment:
Not fixed

Reply:
I do not agree, some languages write "E-mail", in other languages they write without a hyphen, ie "Email", and if I am not mistaken in Sweden they write "E-post" (even in google Translate it says so, but it is known that he cannot always be trusted)
Therefore, it is not the ToDo generator in a given language that gives the information that it is not translated. If the translator adds a given language to the exclusion file, it will be known that it is translated.

Reply:
thx